test/step/launch_test.exs

```diff
@@ -214,6 +238,8 @@ defmodule StepFlow.LaunchTest do
       first_file = "my_file_1.mov"
       source_path = "my_file_2.mov"
       step = @workflow_definition.steps |> List.first()
+      step_name = step.name
+      step_id = step.id
```

Comment:
not needed here, just use it in the call of the method

Reply:
(many times)

Reply:
Corrected with same syntax as the others